src/Translator.java

```diff
@@ -5,15 +5,15 @@
 import java.util.HashMap;
 
 public class Translator {
-   private int  number[];
+   private int  numbers[];
```

Comment:
deze zijn hier niet nodig je maakt de arrays aan in de main classe

Reply:
Klopt, gecorrigeerd